BEST-PRACTICES.md

```diff
@@ -5,6 +5,7 @@ We're glad you're considering contributing to SUSHI! Below are a few best practi
 ## Before contributing
 
 Before contributing a feature or a bugfix, we recommend creating a GitHub issue if one does not exist. This allows the community to provide feedback on why an issue may be occurring or provide additional insight into a suggested feature.
+When creating a GitHub issue, include a description of the feature or bug, any steps needed to demonstrate the issue, and any links to relevant discussion threads on [Zulip](https://chat.fhir.org).
```

Comment:
I think this should have a blank line before it? Or remove the linebreak if you intend it to be all in one paragraph?

Reply:
I had intended to just have it be part of that paragraph, in which case the line break shouldn't change the rendering. But now that I look at it again, I think I like it better as a separate paragraph, so I'll update it.

Reply:
Actually, I kind of changed my mind on this. It felt repetitive, but also not repetitive enough compare to what is in the Contribution Policy section. I decided to just link to that instead and keep it all in one paragraph in [7c4b6fe](https://github.com/FHIR/sushi/pull/1474/commits/7c4b6fee2205e44567d370c0c1b861c3ba6a4746). Let me know what you think of that approach.